.github/workflows/android.yml

```diff
@@ -54,7 +54,6 @@ jobs:
           key: avd-${{ matrix.android-api-level }}
 
       - name: Avd Snapshot
-        if: steps.avd-cache.outputs.cache-hit != 'true'
```

Comment:
Curious why this is getting removed

Reply:
Your curiosity is justified - I just removed this temporarily because the error in the android test said something about "cache needing deletion" and I was just trying whether that helps to resolve it. It does not so I just re-added it.